src/Lunapark/Utils.purs

```diff
@@ -18,12 +19,19 @@ liftAndRethrow a = do
   res ← R.liftAff a
   RE.rethrow res
 
-throwLeft
+throwLeftJsonDecodeError
```

Comment:
Might be worth removing `Left` from this and following function: `throwJsonDecodeError` and `throwLunaparkError` should work

Reply:
added commit `rename throwLeftJsonDecodeError to rethrowAsJsonDecodeError and remove throwLeftUserError`

I think it's better name